app/controllers/users_controller.rb

```diff
@@ -61,7 +61,7 @@ def update
 
   def destroy
     current_user.destroy
-    redirect_to users_url, notice: t('user_was_successfully_deleted')
+    redirect_to users_url, notice: t('user_was_successfully_destroyed')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.